.github/workflows/codeql.yml

```
@@ -32,6 +32,9 @@ jobs:
           - go
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.21.0'
```

```suggestion
          go-version-file: go.mod
```
Let's make it easier to upgrade go later.